src/tools/rbd/action/Kernel.cc

```diff
@@ -153,6 +153,39 @@ static int do_kernel_showmapped(Formatter *f)
   return r;
 }
 
+static void print_error_description(const char *poolname, const char *imgname,
+				    const char *snapname, int maperrno)
+{
+  int r;
+  uint8_t oldformat;
+  librados::Rados rados;
+  librados::IoCtx ioctx;
+  librbd::Image image;
+
+  if (maperrno == ENOENT)
+    return;
```

Comment:
I think we need to print the "see dmesg" message on all errors.  On -ENOENT in particular, since the kernel will output something like "pool rbd does not exist" or "image rbd/b does not exist".